Tasks/HelmDeployV0/task.json

```diff
@@ -12,8 +12,8 @@
     "author": "Microsoft Corporation",
     "version": {
         "Major": 0,
-        "Minor": 1,
-        "Patch": 9
+        "Minor": 137,
```

Comment:
@Anumita the rest of the tasks in the repo use the sprint number as the minor version, so that is what I have done here.